### DISCUSSION TITLE
How to deploy the Promitor docker container step by step

### DISCUSSION THREAD

**Discussion by aftab70** [tomkerkhove/promitor#2068] [General] [answered]
Hi Team,

I am not able to deploy the Promitor docker container on ubuntu machine, Plesse suggest for the same.

sudo docker run -d -p 8999:80 --name promitor-agent-scraper2                          --env PROMITOR_AUTH_APPID='xxxxxxxx'                            --env PROMITOR_AUTH_APPKEY='xxxxxxxxxxxxxx'                          ghcr.io/tomkerkhove/promitor-agent-scraper:latest-linux
Unable to find image 'ghcr.io/tomkerkhove/promitor-agent-scraper:latest-linux' locally
latest-linux: Pulling from tomkerkhove/promitor-agent-scraper
8663204ce13b: Pull complete
a359543aa452: Pull complete
ac3545b2c074: Pull complete
6264dffc94bc: Pull complete
a984834ed340: Pull complete
d5b71be4c764: Pull complete
Digest: sha256:4d43a3e9c57631313a7776e104440b5dfe256724076ddd84ec2d936ff948585a
Status: Downloaded newer image for ghcr.io/tomkerkhove/promitor-agent-scraper:latest-linux
c450fcd3c1dcca0dd56b22b4a61777f8085a970fc3809c5519180851a2cfcc80
aftab70@WebApp:~$ sudo docker ps
CONTAINER ID   IMAGE     COMMAND   CREATED   STATUS    PORTS     NAMES
aftab70@WebApp:~$ sudo docker ps -a
CONTAINER ID   IMAGE                                                     COMMAND                  CREATED          STATUS                      PORTS     NAMES
c450fcd3c1dc   ghcr.io/tomkerkhove/promitor-agent-scraper:latest-linux   "dotnet Promitor.Age…"   13 seconds ago   Exited (4) 12 seconds ago             promitor-agent-scraper2
b6472da7d962   ghcr.io/tomkerkhove/promitor-agent-scraper:2.5.0          "dotnet Promitor.Age…"   4 minutes ago    Exited (4) 4 minutes ago              promitor-agent-scraperrr
6cdd01afd0af   ghcr.io/tomkerkhove/promitor-agent-scraper:2.5.0          "dotnet Promitor.Age…"   7 minutes ago    Exited (1) 7 minutes ago              promitor-agent-scraperr
0748871d9b00   ghcr.io/tomkerkhove/promitor-agent-scraper:2.5.0          "dotnet Promitor.Age…"   10 minutes ago   Exited (1) 10 minutes ago             promitor-agent-scraper
aftab70@WebApp:~$

Container getting down while exaction of the command.

**Comment by tomkerkhove** [comment, marked as answer]
You are not providing a configuration file for the metrics to scrape. If you'll check the logs of the container it should show that.

Every release has a command that should help you get started: https://github.com/tomkerkhove/promitor/releases/tag/Scraper-v2.7.0

If it's no longer up-to-date then I can have a look (but it should)
